backend/src/server/routes/v3/secret-router.ts

```diff
@@ -38,6 +38,13 @@ export const registerSecretRouter = async (server: FastifyZodProvider) => {
     url: "/raw",
     method: "GET",
     schema: {
+      description: "List secrets",
```

Comment:
What does adding `security` and `description` do to the mint docs? I looks like things are already annotated there 


<img width="648" alt="Screenshot 2024-02-12 at 7 32 08 PM" src="https://github.com/Infisical/infisical/assets/9300960/755fbd0c-101a-4f87-b9d2-d2b63ee6d9ec">

Reply:
@maidul98 looks like things are already annotated there. can you tell me the point your referring to

Reply:
This was not their and missing when pointed to prod openapi and makes sense because the other one we added explicitly